pwiz_tools/Skyline/TestPerf/TestPerf.csproj

```diff
@@ -135,7 +135,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="AreaCVHistogramQValueAndRatioTest.cs" />
-    <Compile Include="DdaTutorialTest.cs" />
+    <Compile Include="DdaTutorialTest.cs" />

```

Comment:
Can't see why this is changing.

Reply:
It's a line ending change. 
![image](https://user-images.githubusercontent.com/1200761/97642699-0f5e6280-1a03-11eb-890e-8209d91c8acc.png)

Reply:
The new line ending appears to me to be in error, i.e. just LF and not CRLF?

Reply:
Yes, an error - except that the old one is the erroneous one. The revised/correct one is on the left.